tests/fixtures/sample_lib/modules/primitive/primitive_module.py

```diff
@@ -0,0 +1,52 @@
+# Copyright The Caikit Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A sample module for sample things!
+"""
+# Standard
+from typing import List
+
+# Local
+from ...data_model.sample import PrimitiveTask
+from caikit.core.modules import ModuleLoader
+from caikit.interfaces.common.data_model import StrSequence
+import caikit.core
+
+
+@caikit.core.module(
+    "bd87fbbe-caad-4db6-93f5-37cc3dbedef5", "PrimitiveModule", "0.0.1", PrimitiveTask
+)
+class PrimitiveModule(caikit.core.ModuleBase):
```

Comment:
@HonakerM Does this problem only occur if the `List[T]` types are in the task's set of required parameters? We do have a `sample_lib.modules.sample_task.SamplePrimitiveModule` already which has a bunch of primitive types including `List[T]` and `Union[List[T]...]`, would that one work as is if we called it via the http server?

Reply:
@joerunde I missed that one! It should be testing the same stuff so I'll remove my custom module.